packages/components/src/components/action-pad/action-pad.tsx

```diff
@@ -228,12 +230,54 @@ export class ActionPad extends LitElement {
     this.expandTooltip = tooltips[0];
   }
 
+  private handleToolbarKeyDown(event: KeyboardEvent): void {
```

Comment:
This can be done in a follow-up, but we could extract a controller for these components to use since they have similar logic.

There's also shared logic in the tests, so these could also be refactored as well.